client/tailwind.config.js

```diff
@@ -6,7 +6,15 @@ module.exports = {
     { pattern: /to-(.*)-(500)\/70/ },
   ],
   theme: {
-    extend: {},
+    extend: {
+      backgroundColor: {
+        'slate-light': '#7a8c99',
```

Comment:
Can we rename these to e.g. `bg-slate-450`, so we can have a reference as to how dark it is? And know that it's a background color?